package.json

```diff
@@ -11,12 +11,14 @@
   },
   "devDependencies": {
     "env-cmd": "^10.1.0",
-    "ngrok": "^3.3.0"
+    "ngrok": "^3.3.0",
+    "prettier": "^2.1.2"
```

Comment:
@opensourceally yarn lock file is missing, it should be part of the same PR

Reply:
updated package-lock.json.

Reply:
There is already a yarn.lock file. Use that only

Reply:
Done. Please check now.

Reply:
When you already have yarn.lock, and you have also used it, do you really need package-lock.json?